concourse/scripts/create_concourse_cert.sh

```diff
@@ -61,7 +61,7 @@ if [ "null" = "${dns_validation_record}" ] || [ "null" = "${dns_validation_value
   exit 1
 fi
 
-zone_id=$(aws route53 list-hosted-zones-by-name --dns-name dev.cloudpipeline.digital --query 'HostedZones[0].Id' --output text | cut -d '/' -f 3)
+zone_id=$(aws route53 list-hosted-zones-by-name --dns-name "${ROOT_SYSTEM_DNS_ZONE_NAME}" --query 'HostedZones[0].Id' --output text | cut -d '/' -f 3)
```

Comment:
We already have the zone ID as a Terraform variable, e.g. for dev:

https://github.com/alphagov/paas-bootstrap/blob/190f2bd02783a80b7f35a1e92108eb0f0669451b/terraform/dev.tfvars#L2

Is it possible to use that instead of introducing a new variable?

Reply:
Do you suggest we should parse the tfvars file to get this value?

Reply:
I hadn't thought that far ahead 😆 

It might be easier to pull it out of `*.tfvars` and move it into an environment variable in the `Makefile`, then pass it to Terraform with `-var` like we do for `deploy_env`.

Reply:
I added a second commit which solves this. I created a brand new deployment and then destroyed it successfully.